src/views/Settings/PowerRestorePolicy/PowerRestorePolicy.vue

```diff
@@ -1,131 +1,123 @@
 <template>
-  <b-container fluid="xl">
+  <BContainer fluid="xl">
     <page-title
       :title="$t('appPageTitle.powerRestorePolicy')"
       :description="$t('pagePowerRestorePolicy.description')"
     />
-
-    <b-row>
-      <b-col>
+    <BRow>
+      <BCol>
         <alert v-if="isOperatingModeManual" variant="warning" class="mb-5">
-          <b-row align-v="center">
-            <b-col>
+          <BRow align-v="center">
+            <BCol>
               <p class="mb-0">
                 {{ $t('pagePowerRestorePolicy.alert.manualOperatingMode') }}
               </p>
-            </b-col>
-            <b-col>
+            </BCol>
+            <BCol>
               <div>
-                <b-link to="/operations/server-power-operations">
+                <BLink to="/operations/server-power-operations">
                   {{ $t('pagePowerRestorePolicy.alert.changeServerOpMode') }}
-                </b-link>
+                </BLink>
               </div>
-            </b-col>
-          </b-row>
+            </BCol>
+          </BRow>
         </alert>
-      </b-col>
-    </b-row>
-    <b-row>
-      <b-col sm="8" md="6" xl="12">
-        <b-form-group :label="$t('pagePowerRestorePolicy.powerPoliciesLabel')">
-          <b-form-radio-group
+      </BCol>
+    </BRow>
+    <BRow>
+      <BCol sm="8" md="6" xl="12">
+        <BFormGroup :label="$t('pagePowerRestorePolicy.powerPoliciesLabel')">
+          <BFormRadioGroup
             v-model="currentPowerRestorePolicy"
             :disabled="isOperatingModeManual"
             :options="options"
             name="power-restore-policy"
             stacked
-          ></b-form-radio-group>
-        </b-form-group>
-      </b-col>
-    </b-row>
+          ></BFormRadioGroup>
+        </BFormGroup>
+      </BCol>
+    </BRow>
 
-    <b-button
+    <BButton
```

Comment:
There is no gap between the options and the Save button. Please make this change